packages/nx/src/plugins/js/index.ts

```diff
@@ -51,7 +52,10 @@ export const createNodes: CreateNodes = [
     }
 
     const lockFilePath = join(workspaceRoot, lockFile);
-    const lockFileContents = readFileSync(lockFilePath).toString();
+    const lockFileContents =
+      packageManager !== 'bun'
+        ? readFileSync(lockFilePath).toString()
+        : execSync('bun bun.lockb').toString();
```

Comment:
```suggestion
        : execSync(`bun ${lockFilePath}`).toString();
```

Reply:
Please apply the same fix on line 101.

Reply:
Done that. Sorry for the confusion; i only knew about self-excuting on linux

Reply:
Optionally we can use ``execSync(`bun ${lockFilePath}`, { encoding: 'utf8' })`` to ensure proper decoding while stringification.